### ISSUE TITLE
Path completion in interactive windows does not return current working directory

### ISSUE THREAD

**Issue by stevenlis** [microsoft/vscode-jupyter#14253]
## Environment data

Version: 1.81.1
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:40:25.698Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0
Jupyter: v2023.7.1002162226

## Expected behaviour

Path completion behavior is the same in both the editor and the interactive window.

## Actual behaviour

Path completion in the interactive window does not recognize the current working directory but works fine in the editor. 

I have tried to disable all my path completion extensions in an attempt to resolve the recurring issue.

<img width="1189" alt="CS-z7SCJoWS@2x" src="https://github.com/microsoft/vscode-jupyter/assets/33796896/9e229a98-42af-4fae-a651-430e9ea55777">

<img width="1186" alt="CS-PNUg6Lnt@2x" src="https://github.com/microsoft/vscode-jupyter/assets/33796896/3ad7c855-22c0-4a4a-9a14-db33d397db7b">

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. XXX

## Logs

<details>

<summary>Output for <code>Jupyter</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Jupyter</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

**Comment by endlessbeloved** [comment]
I also meet a "file was not found" in interactive windows. 
I suppose the cause reason could be interactive windows trying to find file in root directory path( /${working_dir}/file.path) rather than current working directory(${working_dir}/file.path)
#14225

**Comment by stevenlis** [comment]
@amunger Hi, is there any update on this issue?

**Comment by amunger** [comment]
which language server are you using for intellisense? I get path completions in .py files with jedi (not pylance), but I don't get completions at all in the IW input box.

**Comment by amunger** [comment]
actually, I get them for an ubuntu machine with jedi, and it lists files in the workspace correctly.

**Comment by stevenlis** [comment]
@amunger I actually do use pylance.

**Comment by stevenlis** [comment]
> which language server are you using for intellisense? I get path completions in .py files with jedi (not pylance), but I don't get completions at all in the IW input box.

I experience the same behavior as shown in the screenshots, and I use Pylance.

**Comment by amunger** [comment]
@debonte - do you know how the root directory is being retrieved/set for the IW input box in Pylance?

**Comment by debonte** [comment]
I don't believe that Pylance supports path completions. @heejaechang would know for sure though.

**Comment by DonJayamanne** [comment]
Completions are returned by the kernel, 
If this isn't returned by the kernel, then there's no plan to explicitly return this in Jupyter extension.
